Syntaxes/Nim CFG.sbnf

```diff
@@ -0,0 +1,218 @@
+EXTENSIONS = 'nim.cfg'
```

Comment:
Is this file meant to be used with any specific tool, or is it just for reference?

Reply:
https://github.com/BenjaminSchaaf/sbnf

Reply:
Hm, I'm going to have to think on this. I'm not sure about adding another dependency to this package's development.

Reply:
Hm, I'm going to have to think on this. I'm not sure about adding another dependency to this package's development.